india_compliance/gst_india/utils/exporter.py

```
@@ -27,6 +30,15 @@ def create_sheet(self, **kwargs):
 
         Worksheet().create(workbook=self.wb, **kwargs)
 
+    def insert_data(self, sheet_name, fields, data, start_row=1, start_column=1):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)